Assets/PC2D/Scripts/PlatformerMotor2D.cs

```diff
@@ -3321,5 +3275,155 @@ void OnDrawGizmosSelected()
         }
     }
 
+    private void ChangeState(MotorState newState) {

```

Comment:
I did some Debug.Log while developing, found no problem.
I saw Unity Testing tools, maybe we can use them... in a near future, prior the big Mode refactoring maybe...